ccvm_simulators/post_processor/grad_descent.py

```diff
@@ -0,0 +1,49 @@
+from .post_processor import PostProcessor
+import time
+import torch
+import tqdm
+
+
+class PostProcessorGradDescent(PostProcessor):
+    def __init__(self, num_iter_main):
```

Comment:
What does `num_iter_main` represent? We should document this parameter.
A bigger issue, though, is that our example flow doesn't currently support passing parameters to post processors (try running `ccvm_boxqp_dl.py` with this post processor). We will need to figure something out; it might require us to restructure the flow.

Reply:
https://github.com/1QB-Information-Technologies/ccvm/issues/39
This issue is relevant.

Reply:
Got it. I added the doc for `num_iter_main`. The #39 issue will be addressed in another PR.

Reply:
The issue is that this class is currently unusable in the way that we expect users to interact with it.
I think a good intermediate fix could be to 
* remove `num_iter_main` for now and just default the value
* make a new ticket to put it back in

**order of operations:** update and merge this PR, address #39 , then address the new ticket.

Reply:
Done.
- Removed `num_iter_main` in the constructor class and put back in the `postprocess()` with a default value `100`. I've also sent a message to confirm the default value with @FarhadK-1QBit and will update it if necessary.
- New ticket created. #156